fenics/environment.yml

```
@@ -0,0 +1,9 @@
+channels:
+- conda-forge
+dependencies:
+  - fenics =2019.1.0
+  - fenics-libdolfin =2019.1.0=*_8
+  - mshr =2019.1.0=*_2
```

```suggestion
  - mshr =2019.1.0
```
Based on https://github.com/pyiron/pyiron_contrib/pull/42 it is no longer necessary to fix a specific build number